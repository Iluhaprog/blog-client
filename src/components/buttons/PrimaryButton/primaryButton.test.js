import React from 'react';
import { render } from '@testing-library/react';
import { PrimaryButton } from '.';

test('Test view of danger button', () => {
    const { asFragment } = render(<PrimaryButton text='test'/>);
    expect(asFragment()).toMatchSnapshot();
});