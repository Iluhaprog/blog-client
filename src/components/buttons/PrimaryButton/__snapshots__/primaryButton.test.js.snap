// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test view of danger button 1`] = `
<DocumentFragment>
  <button
    class="button default-button primary-button"
  >
    test
  </button>
</DocumentFragment>
`;
